### PR TITLE
New conformity page

### PR DESCRIPTION
**Pull request by JamesH65**
Added a page to link to the published conformity documents

### CODE REVIEW

**Comment by LornaLynch** [comment]
I have this now

**Comment by LornaLynch** [comment]
Looks good: merging

File: hardware/raspberrypi/conformity.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - Electromagnetic Compatibility Directive (EMC) 2014/30/EU,
+  - Restriction of Hazardous Substances (RoHS) Directive 2011/65/EU
+  - EN55032:2012 Class B
+
+- USA
+  - ANSI C63.4 
+  - FCC 47 CFR Part 15, Subpart B, Class B Digital Device.
+
+ [FCC Conformance Document](https://www.raspberrypi.org/files/compliance/RaspberryPiCM3_DOC_FCC.pdf)   
+ [EU conformance Document](https://www.raspberrypi.org/files/compliance/RaspberryPiCM3_DOC_FCC.pdf)
```
Comment (lurch) [inline hardware/raspberrypi/conformity.md]: These both link to the same PDF - is that correct?
Perhaps the second link should be to https://www.raspberrypi.org/files/compliance/RaspberryPiCM3_DOC_EU.pdf

Comment (LornaLynch) [inline hardware/raspberrypi/conformity.md]: I can change it if that's better?

Comment (LornaLynch) [inline hardware/raspberrypi/conformity.md]: Done: thanks @lurch

**Comment by JamesH65** [comment]
Sorry, C&P error when writing the document - thanks for sorting it out. I
blame working form a laptop whilst ill.

On 5 June 2017 at 16:51, Lorna Lynch <notifications@github.com> wrote:

> *@LornaLynch* commented on this pull request.
> ------------------------------
>
> In hardware/raspberrypi/conformity.md
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_raspberrypi_documentation_pull_658-23discussion-5Fr120140314&d=DwMCaQ&c=DpyQ_ftY536pf7wCBQXXU58xADDRY77THQzJu1OmzOo&r=w09_2ePv8G3zRjoV19Wm1Q6rI7CDlOns4PuRv2hHkek&m=hWeQckNO62cfU_VP52QgGs-wr9Yv5LHkT6b15kJZYhY&s=85M49zoIuXFpJGSSLuFANEvhOPnyiBMifODFa0w2w3g&e=>
> :
>
> > +
> +## Compute Module 3 and Compute Module 3 Lite
> +
> +All Raspberry Pi Compute Module 3 devices and the IO board have undergone extensive compliance testing and meet the following standards.
> +
> +- EU
> +  - Electromagnetic Compatibility Directive (EMC) 2014/30/EU,
> +  - Restriction of Hazardous Substances (RoHS) Directive 2011/65/EU
> +  - EN55032:2012 Class B
> +
> +- USA
> +  - ANSI C63.4
> +  - FCC 47 CFR Part 15, Subpart B, Class B Digital Device.
> +
> + [FCC Conformance Document](https://www.raspberrypi.org/files/compliance/RaspberryPiCM3_DOC_FCC.pdf)
> + [EU conformance Document](https://www.raspberrypi.org/files/compliance/RaspberryPiCM3_DOC_FCC.pdf)
>
> Done: thanks @lurch
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_lurch&d=DwMCaQ&c=DpyQ_ftY536pf7wCBQXXU58xADDRY77THQzJu1OmzOo&r=w09_2ePv8G3zRjoV19Wm1Q6rI7CDlOns4PuRv2hHkek&m=hWeQckNO62cfU_VP52QgGs-wr9Yv5LHkT6b15kJZYhY&s=iEzRfZa7o6393SzHJ20eq4RylCKKNfaWzyB-n-dJzx4&e=>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_raspberrypi_documentation_pull_658-23discussion-5Fr120140314&d=DwMCaQ&c=DpyQ_ftY536pf7wCBQXXU58xADDRY77THQzJu1OmzOo&r=w09_2ePv8G3zRjoV19Wm1Q6rI7CDlOns4PuRv2hHkek&m=hWeQckNO62cfU_VP52QgGs-wr9Yv5LHkT6b15kJZYhY&s=85M49zoIuXFpJGSSLuFANEvhOPnyiBMifODFa0w2w3g&e=>,
> or mute the thread
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ADqrHUoROHVaqHCLuBhEqTODTL92kVceks5sBCQUgaJpZM4NwFLp&d=DwMCaQ&c=DpyQ_ftY536pf7wCBQXXU58xADDRY77THQzJu1OmzOo&r=w09_2ePv8G3zRjoV19Wm1Q6rI7CDlOns4PuRv2hHkek&m=hWeQckNO62cfU_VP52QgGs-wr9Yv5LHkT6b15kJZYhY&s=dhcPpX5jAcm6lgVYtBNNYGqOJneudN_Si4nf3S633Xc&e=>
> .
>



-- 
James Hughes
Principal Software Engineer,
Raspberry Pi (Trading) Ltd

**Pull request merged**